### PR TITLE
DataGrid - using DropDownButton in export toolbar button

### PR DESCRIPTION
**Pull request by pomahtri**
Updated etalons: https://github.com/DevExpress/devextreme-demos/pull/1206

---

I didn't know this, so I'll mention it here: you can compare screenshots using this button: 
image

### CODE REVIEW

File: js/ui/data_grid/ui.data_grid.export.js [hunk 1]
```diff
@@ -699,127 +687,95 @@ dataGridCore.registerModule('export', {
...
+                            displayExpr: 'text',
+                            items: items,
+                            hint: 'Export',
+                            elementAttr: {
+                                class: 'dx-datagrid-export-button'
+                            },
+                            dropDownOptions: {
+                                wrapperAttr: { 'class': 'dx-datagrid-export-menu' },
+                                width: 'auto'
+                            }
```
Comment (NickMitrokhin) [inline js/ui/data_grid/ui.data_grid.export.js]: It is better to move class names to constants.

File: js/ui/data_grid/ui.data_grid.export.js [hunk 2]
```diff
@@ -876,12 +796,12 @@ dataGridCore.registerModule('export', {
                     if(allowExportSelected) {
                         options = {
                             hint: texts.exportTo,
-                            icon: DATAGRID_EXPORT_ICON
+                            icon: 'export-to'
```
Comment (NickMitrokhin) [inline js/ui/data_grid/ui.data_grid.export.js]: Why did you remove the DATAGRID_EXPORT_ICON constant?

File: js/ui/data_grid/ui.data_grid.export.js [hunk 3]
```diff
@@ -876,12 +796,12 @@ dataGridCore.registerModule('export', {
...
                         };
                     } else {
                         options = {
                             hint: texts.exportAll,
-                            icon: DATAGRID_EXPORT_EXCEL_BUTTON_ICON,
+                            icon: 'export-excel-button',
```
Comment (NickMitrokhin) [inline js/ui/data_grid/ui.data_grid.export.js]: Why did you remove the DATAGRID_EXPORT_EXCEL_BUTTON_ICON constant?

File: js/ui/data_grid/ui.data_grid.export.js [hunk 4]
```diff
@@ -699,127 +687,95 @@ dataGridCore.registerModule('export', {
...
-                                    that._renderList(data, $(container));
-                                },
-                                name: 'exportButton',
-                                allowExportSelected: true,
-                                location: 'after',
-                                locateInMenu: 'auto',
-                                sortIndex: 30
-                            });
+                    const toolbarButtonOptions = {
+                        name: 'exportButton',
```
Comment (NickMitrokhin) [inline js/ui/data_grid/ui.data_grid.export.js]: Isn't it better to extract the predefined item name to a constant?

**Review by vconst — APPROVED** [review]

**Review by NickMitrokhin — APPROVED** [review]
